app/src/main/java/ch/epfl/sdp/peakar/database/DatabaseSnapshot.java

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.sdp.peakar.database;
+
+import androidx.annotation.NonNull;
+
+import com.google.firebase.database.DataSnapshot;
+
+import java.util.Iterator;
+
+/**
+ * This class represents a Database Snapshot, i.e. a copy of a database child retrieved and available locally.
+ * Its implementation makes use of Firebase API, so needs to be modified if database provider is changed.
+ */
+public class DatabaseSnapshot {

```

Comment:
Javadocs comments on the functions would be nice. 
Even if the methods only call the firebase methods, it is nice to pass the mouse pointer on a function a see the comments while using one of those methods

Reply:
Yeah sorry, I forgot to add doc to this class after I finished implementing and testing it.
Thanks for letting me notice it.